packages/ui-library/tailwind.config.js

```diff
@@ -2,4 +2,8 @@
 module.exports = {
 	presets: [ require( "@yoast/tailwindcss-preset" ) ],
 	content: [ "./src/**/*.js" ],
+	safelist: [
+		"yst-grow",
+		"yst-gap-6",
+	],
```

Comment:
Provide tailwind to storybook instead?
https://storybook.js.org/recipes/tailwindcss#2-provide-tailwind-to-storieshttps://storybook.js.org/recipes/tailwindcss#2-provide-tailwind-to-stories